src/components/MostPopular/ProductItem/__snapshots__/ProductItem.spec.js.snap

```diff
@@ -43,7 +43,7 @@ exports[`<ProductItem /> should render component <ProductItem /> 1`] = `
     <h3
       className="product__title-wrapper__price"
     >
-      $599
+      NaN
```

Comment:
This NaN looks suspicious, have you tested the price rendering on the Product Item?